src/main/resources/static/admin/adminjs/jquery.equalHeight.js

```diff
@@ -0,0 +1,20 @@
+// make sure the $ is pointing to JQuery and not some other library
+    (function($){
+        // add a new method to JQuery
+
+        $.fn.equalHeight = function() {
+           // find the tallest height in the collection
+           // that was passed in (.column)
+            tallest = 0;
+            this.each(function(){
+                thisHeight = $(this).height();
```

Comment:
![BLOCKER](https://sonarsource.github.io/sonar-github/severity-blocker.png 'Severity: BLOCKER') Add the "let", "const" or "var" keyword to this declaration of "thisHeight" to make it explicit. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.com/coding_rules#rule_key=javascript%3AS2703)